charts/rclone-copy/templates/rclone-cron.yaml

```diff
@@ -33,6 +33,10 @@ spec:
             {{ $key }}: {{ $value }}
             {{- end }}
         spec:
+          {{- if .Values.tolerations }}
+          tolerations:
+{{ toYaml .Values.tolerations | indent 12 }}
```

Comment:
Did you check if the indents are correct for all 3 charts?

Reply:
Yes, now I checked all of them